packages/twig/src/patterns/components/card/card.twig

```diff
@@ -0,0 +1,95 @@
+{#
+  CARD COMPONENT
+#}
+<script>console.log({{ _context | json_encode | raw }});</script>
+<div class="{{prefix}}--card {{prefix}}--card--{{theme}} {{prefix}}--card--{{variant}}{% if cornercut %} {{prefix}}--card--{{cornercut}}{% endif %}{% if color %} {{prefix}}--card--{{color}}{% endif %}{% if alignment %} {{prefix}}--card--{{alignment}}{% endif %} {% if link %} {{prefix}}--card--action{% endif %}">
+  {% if link %}
+    <a class="{{prefix}}--card--link" href="{{link}}" title="{{title}}"></a>
+  {% endif %}
+  <div class="{{prefix}}--card--wrap">
+    {% if image %}
+      <div class="{{prefix}}--card--image--wrapper"><picture><img class="{{prefix}}--card--image" src="{{image}}" alt="{{title}}"></picture></div>
+    {% endif %}
+    <div class="{{prefix}}--card--content">
+      {% if eyebrow %}
+        <p class="{{prefix}}--card--eyebrow">{{eyebrow}}</p>
+      {% endif %}
+      {% if title %}
+        <h5 class="{{prefix}}--card--title">{{title}}</h5>
+      {% endif %}
+      {% if intro %}
+        <p class="{{prefix}}--card--intro">{{intro}}</p>
+      {% endif %}
+      {% if date %}
```

Comment:
Can you make this the `<time>` tag and also output the UNIX `datetime` attribute?

Reply:
Yes good point.